user-access-manager.php

```diff
@@ -132,9 +132,6 @@ function userAccessManagerAP()
         ) {
             //Admin actions
            if (function_exists('add_action')) {
-                add_action('admin_print_styles', array($oUserAccessManager, 'addStyles'));
-                add_action('wp_print_scripts', array($oUserAccessManager, 'addScripts'));
```

Comment:
the previous comment was just for documentation purposes. The patch should be complete like it is.